src/app/Login/page.module.css

```diff
@@ -0,0 +1,56 @@
+.EmailContainer {
+  align-items: center;
+  width: 324px;
+  height: 48px;
+  margin-bottom: 80px;
+}
+.PasswordContainer {
+  align-items: center;
+  width: 324px;
+  height: 48px;
+  margin-bottom: 140px;
+}
+.label {
+  margin-bottom: 10px;
+  display: flex;
+}
+.span {
+  margin-top: 20px;
+  display: flex;
+  margin-bottom: 20px;
+  justify-content: center;
+}
+
+.welcomeBackcontainer {
+  margin: 100px;
+  text-align: center;
+  font-size: 50px;
+  text-align: center;
+  color: #ffffff;
+  margin-bottom: 100px;
+}
+
+.loginPageContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
+  width: 100vw;
+}
+
+.lgnbttncontainer,
+.signbttncontainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 20px;
+}
+
+.Err {
+  color: #e74c3c;
+  font-size: 14px;
+  margin-top: 5px;
+  font-weight: bold;
+  display: flex;
+  align-items: center;
+}
```

Comment:
define the error class at the textbox component so that way all text fields can have this error styling no matter where its imported from. currently this styling will only exist in the login page and we don't want that